src/slaptcurl.c

```diff
@@ -60,9 +60,9 @@ int slapt_download_data(FILE *fh, const char *url, size_t bytes, long *filetime,
 
     if (!global_config->dl_stats) {
         if (global_config->progress_cb == NULL) {
-            curl_easy_setopt(ch, CURLOPT_PROGRESSFUNCTION, slapt_progress_callback);
+            curl_easy_setopt(ch, CURLOPT_XFERINFOFUNCTION, slapt_progress_callback);
```

Comment:
It would be nice if this were merged. Right now the build fails on Slackware 15.0

```
../src/slaptcurl.c:63:13: error: ‘CURLOPT_PROGRESSFUNCTION’ is deprecated: since 7.32.0. Use CURLOPT_XFERINFOFUNCTION [-Werror=deprecated-declarations]
```

I've tested the PR with https://github.com/andy5995/docker-slackware-build-essential (I needed to install libgpg-error and libassuan) and it builds successfully.

Thanks @jaos

Reply:
Yep, apologies I've been doing other stuff over the holiday. Will complete.

Reply:
> Yep, apologies I've been doing other stuff over the holiday. Will complete.

No apologies necessary @jaos (I completely understand)! Happy New Year...